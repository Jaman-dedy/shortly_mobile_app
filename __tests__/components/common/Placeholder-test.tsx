import React from 'react';
import renderer from 'react-test-renderer';
import {cleanup} from '@testing-library/react-native';

import Placeholder from '../../../src/components/common/PlaceHolder';

import MockThemes from '../../../mock/MockTheme';

afterEach(cleanup);
test('renders correctly', async () => {
  const tree = await renderer
    .create(
      <MockThemes>
        <Placeholder />
      </MockThemes>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
