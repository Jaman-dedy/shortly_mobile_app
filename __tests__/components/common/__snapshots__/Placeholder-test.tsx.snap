// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  onLayout={[Function]}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "column",
      }
    }
  >
    <View
      style={Object {}}
    >
      <View
        style={
          Object {
            "position": "relative",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "borderRadius": 4,
                "height": 20,
                "width": 320,
              },
              Object {
                "backgroundColor": "#E1E9EE",
                "overflow": "hidden",
              },
            ]
          }
        />
      </View>
      <View
        style={
          Object {
            "position": "relative",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "borderRadius": 4,
                "height": 30,
                "marginTop": 6,
                "width": 320,
              },
              Object {
                "backgroundColor": "#E1E9EE",
                "overflow": "hidden",
              },
            ]
          }
        />
      </View>
    </View>
  </View>
</View>
`;
